lib/Qi.pm

```diff
@@ -13,6 +20,10 @@ sub startup {
 
   # Normal route to controller
   $r->get('/')->to('example#welcome');
+
+  # Migrate to latest version if necessary
+  my $path = $self->home->rel_file('migrations/Qi.sql');
```

Comment:
Use lower case file names.  qi.sql.  Update here and rename the file on disk.

Why?  All projects need consistency and style guidelines etc.  We haven't developed those guidelines and whatnot, so I'll just direct you as we go.  I base my decisions subjectively on years of experience and exposure to others' styles.